.travis.yml

```diff
@@ -24,6 +24,8 @@ matrix:
     include:
         - php: 7.0
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci"
+    allow_failures:
+        - php: hhvm
```

Comment:
Maybe add fast_forward as well, so that Travis doesn't wait for the allowed to fail builds to announce the result.